src/main/java/pl/asie/computronics/integration/gregtech/gregtech5/DriverTTMultiblockBase.java

```diff
@@ -0,0 +1,55 @@
+package pl.asie.computronics.integration.gregtech.gregtech5;
+
+import net.minecraft.tileentity.TileEntity;
+import net.minecraft.world.World;
+import net.minecraftforge.common.util.ForgeDirection;
+
+import gregtech.api.metatileentity.BaseMetaTileEntity;
+import li.cil.oc.api.machine.Arguments;
+import li.cil.oc.api.machine.Callback;
+import li.cil.oc.api.machine.Context;
+import li.cil.oc.api.network.ManagedEnvironment;
+import li.cil.oc.api.prefab.DriverSidedTileEntity;
+import pl.asie.computronics.integration.ManagedEnvironmentOCTile;
+import tectech.thing.metaTileEntity.multi.base.Parameters;
+import tectech.thing.metaTileEntity.multi.base.TTMultiblockBase;
+
+public class DriverTTMultiblockBase extends DriverSidedTileEntity {
+
+    public static class ManagedEnvironmentTTMultiblockBase extends ManagedEnvironmentOCTile<BaseMetaTileEntity> {
+
+        public ManagedEnvironmentTTMultiblockBase(BaseMetaTileEntity tile, String name) {
+            super(tile, name);
+        }
+
+        @Callback(doc = "function(hatch:int, id: int, val:double); Sets a parameter", direct = true)
```

Comment:
This shouldn't be `direct` since writing parameters is likely not thread-safe

(Also could you fix the spacing on `id: int` to be like the others)